src/js/experiments/gettingStartedGuide.js

```diff
@@ -23,30 +25,43 @@ function showHomePageBadges() {
   }
 }
 
-function setUpTracking() {
-  const badges = Array.from($('.wrapper-link'));
-  badges.forEach((badge) => {
-    badge.addEventListener('click', () => {
-      window.AnalyticsClient.trackAction('clicked-landing-page-badge', {
-        badgeText: badge.innerText,
-        badgeHref: badge.href,
+function setUpTracking(variation) {
+  const newBadges = $("li > a[href='/docs/2.0/getting-started/']");
+  if (newBadges) {
+    const sidebarBadge = newBadges[1];
+    sidebarBadge.addEventListener('click', () => {
+      window.AnalyticsClient.trackAction('clicked-sidebar-getting-started', {
+        variation,
       });
     });
-  });
-  const links = Array.from($('.treatment').find($('a')));
-  links.forEach((link, i) => {
-    link.addEventListener('click', () => {
-      window.AnalyticsClient.trackAction(
-        'clicked-on-getting-started-guide-link',
-        {
-          linkText: link.innerText,
-          linkHref: link.href,
-          linkIndexOnPage: i + 1 + '/' + links.length,
-          page: window.location.pathname,
-        },
-      );
+  }
+
+  // Since we are using this function for both variations, we only want to add tracking for elements if we are in treatment as they are unique for the experiment
+  if (variation === 'treatment') {
+    const badges = Array.from($('.wrapper-link'));
+    badges.forEach((badge) => {
+      badge.addEventListener('click', () => {
+        window.AnalyticsClient.trackAction('clicked-landing-page-badge', {
+          badgeText: badge.innerText,
+          badgeHref: badge.href,
+        });
+      });
     });
-  });
+
+    const links = Array.from($('.treatment').find($('a')));
+    links.forEach((link, i) => {
+      link.addEventListener('click', () => {
+        window.AnalyticsClient.trackAction(
+          'clicked-on-getting-started-guide-link',
+          {
+            linkText: link.innerText,
+            linkIndexOnPage: i + 1 + '/' + links.length,
+            page: window.location.pathname,
+          },
+        );
+      });
+    });
+  }
```

Comment:
do we have multiple links?

Reply:
Yeah I think around 20 on the page when in treatment